app/src/main/java/com/pvh98/calculator/CalculatorFrg.kt

```diff
@@ -0,0 +1,217 @@
+            }
+
+            R.id.bt_seven -> {
+                onDigit("7")
+            }
+
+            R.id.bt_eight -> {
+                onDigit("8")
+            }
+
+            R.id.bt_nine -> {
+                onDigit("9")
+            }
+
+            R.id.bt_zero -> {
+                onDigit("0")
+            }
+
+            R.id.bt_dot -> {
+                onDecimal()
+            }
+
+            R.id.bt_deleteAll -> {
+                onClear()
+            }
+
+            R.id.bt_delete -> {
+                onDelete()
+            }
+
+            R.id.bt_add -> {
+                if (tv_result.text.toString().length != 0) {
+                    mOpeAdd = true
+                    onOpeDecimal = false
+                    onValueOne()
+                }
+            }
+
+            R.id.bt_minus -> {
+                if (tv_result.text.toString().length != 0) {
+                    mOpeSub = true
+                    onOpeDecimal = false
+                    onValueOne()
+                }
+            }
+
+            R.id.bt_mutiple -> {
+                if (tv_result.text.toString().length != 0) {
+
+
+                    mOpeMul = true
+                    onOpeDecimal = false
+                    onValueOne()
+                }
+            }
+
+            R.id.bt_divide -> {
+                if (tv_result.text.toString().length != 0) {
+                    mOpeDiv = true
+                    onOpeDecimal = false
+                    onValueOne()
+                }
+            }
+
+            R.id.bt_euclidDivide -> {
+                if (tv_result.text.toString().length != 0) {
+                    mOpeEucDiv = true
+                    onOpeDecimal = false
+                    onValueOne()
+                }
+            }
+
+            R.id.bt_equal -> {
+
+                if (mOpeAdd) {
+                    mValueTwo = tv_result.text.toString().toFloat()
+                    tv_result.text = (mValueOne + mValueTwo).toString()
+                    mOpeAdd = false
+                }
+                if (mOpeSub) {
+                    mValueTwo = tv_result.text.toString().toFloat()
+                    tv_result.text = (mValueOne - mValueTwo).toString()
+                    mOpeSub = false
+                }
+                if (mOpeMul) {
+                    mValueTwo = tv_result.text.toString().toFloat()
+                    tv_result.text = (mValueOne * mValueTwo).toString()
+                    mOpeMul = false
+                }
+                if (mOpeDiv) {
+                    mValueTwo = tv_result.text.toString().toFloat()
+                    tv_result.text = (mValueOne / mValueTwo).toString()
+                    mOpeDiv = false
+                }
+                if (mOpeEucDiv) {
+                    tv_result.text = (mValueOne % mValueTwo).toString()
+                    mOpeEucDiv = false
+                }
+            }
+        }
+    }
+
+    private fun onDecimal() {
```

Comment:
```suggestion
    private fun enterDecimal() {
```
tên hàm nên bắt đầu bằng động từ